test/e2e/volume_prune_test.go

```diff
@@ -150,6 +150,15 @@ var _ = Describe("Podman volume prune", func() {
 		Expect(session).Should(Exit(0))
 		Expect(session.OutputToStringArray()).To(HaveLen(3))
 
+		session = podmanTest.Podman([]string{"volume", "create", "--label", "testlabel", "myvol5"})
```

Comment:
```suggestion
		session = podmanTest.Podman([]string{"volume", "create", "--label", "testlabel", "myvol7"})
```

Volume name already takes in step above, either that volume must be removed or this volume gotta have a new name